unittest/test_fastpfor.cpp

```diff
@@ -116,7 +116,7 @@ namespace FastPForLib {
           std::uniform_int_distribution<int32_t> dist(
                                   std::numeric_limits<int32_t>::min(),
                                   std::numeric_limits<int32_t>::max());
-          for (int i = 0; i < values; ++i) {
+          for (uint32_t i = 0; i < values; ++i) {
```

Comment:
`unsigned`. Even though values is `uint32_t`, it's quite strange to use uint32_t as loop counter.
same for a change bellow

Reply:
Seems necessary to ensure `i < values` is both efficient and correct for `values` approaching `std::numeric_limits<uint32_t>::max()` right?

Reply:
Maybe to avoid any concern, you could use the standard algorithms library to fill the vector in a declarative fashion.

```suggestion
          std::generate_n(std::begin(v), values, std::bind(dist, e2));
```

Reply:
<em>it's quite strange to use uint32_t as loop counter.</em>

Why? Many compilers object to signed/unsigned comparisons. I don't want to change the type of `values`. The alternative for the counters are `size_t`, `unsigned int`, ... none of them functionally better. Well, `size_t` (an unsigned type) is arguably better but I am not certain it matters.

> std::generate_n(std::begin(v), values, std::bind(dist, e2));

@cbsmith Good idea, but it is out of scope here. I only changed that bit to silence a warning. If you'd like to modernize the C++, the please consider issue a distinct PR. There is much that could be done...